cmd/litestream/main.go

```diff
@@ -172,6 +174,14 @@ type Config struct {
 
 	// Logging
 	Logging LoggingConfig `yaml:"logging"`
+
+	// MCP server options
+	MCPEnabled bool `yaml:"mcp-enabled"`
+	MCPPort    int  `yaml:"mcp-port"`
```

Comment:
Can you make this similar to the `Addr` field and just have an `MCPAddr` that is enabled if set? I prefer that over "port" so it can fully set the bind address.